modules/social_features/social_event/config/schema/social_event.schema.yml

```diff
@@ -11,3 +11,31 @@ social_event.settings:
     disable_event_enroll:
       type: boolean
       label: 'Disable all event enrollments on your community'
+    show_user_timezone:
+      type: boolean
+      label: 'Show user timezone'
+    langcode:
+      type: string
+      label: 'Language code'
+    _core:
+      type: _core_config_info
```

Comment:
The `_core` config info doesn't need a type I believe. This should be handled entirely by core.

Reply:
Hmm, that is interesting because I faced this error during running the test, but now I tested it without this `_core` and it works, so, yeah, definitely we can remove it.

Reply:
I checked related issues on drupal.org and it seems that they are not fixed within this PR.